scheduler/src/cook/kubernetes/api.clj

```diff
@@ -810,6 +839,16 @@
         ; gpu count is not stored in scalar-requests because Fenzo does not handle gpus in binpacking
         gpus (or (:gpus resources) 0)
         gpu-model-requested (constraints/job->gpu-model-requested gpus job pool-name)
+        ; if disk config for pool has enable-constraint? set to true, add disk info to the job
+        enable-disk-constraint? (regexp-tools/match-based-on-pool-name (config/disk) pool-name :enable-constraint? :default-value false)
+        ; if user did not specify disk request, use default on pool
+        disk-request (when enable-disk-constraint?
+                       (or (-> resources :disk :request)
+                           (regexp-tools/match-based-on-pool-name (config/disk) pool-name :default-request)))
+        disk-limit (when enable-disk-constraint? (-> resources :disk :limit))
+        ; if user did not specify disk type, use default on pool
+        disk-type (when enable-disk-constraint? (constraints/job->disk-type-requested (-> resources :disk :type) pool-name))
```

Comment:
yep, done

Reply:
I think it would be better to leave these lines here because outside of this `let` block, some of the variables like `resources` get changed. I actually tried to move these lines down to where they were consumed but was not able to get the correct disk specs set on the job

Reply:
I'm ok with these here

Reply:
Sounds good